ansible/fcrepo4.yml

```diff
@@ -1,6 +1,16 @@
 ---
+- name: dependency provisioning
+  hosts: all
+  become: yes
+  become_method: sudo
+  gather_facts: false
+  tasks:
+    - name: install python2
```

Comment:
If you are installing Python 2 just to get Ansible to work to provision on a Xenial host then that is not necessary.  The [recommended way](https://docs.ansible.com/ansible/latest/reference_appendices/python_3_support.html#using-python-3-on-the-managed-machines-with-commands-and-playbooks) is to set the `ansible_python_interpreter` Ansible variable to `/usr/bin/python3`.

In Vagrant terms, this means you can add the following to your `config.vm.provision :ansible` section:
```
ansible.host_vars = {
   :fcrepo4 => {'ansible_python_interpreter' => '/usr/bin/python3'}
}
```

Reply:
I am curious to know: what "other errors" would the above change introduce?

All currently supported versions of Ansible (2.5 onwards) support Python 3, and the above method is the recommended way of telling Ansible to use Python 3 on hosts that have only Python 3 installed.

If there is other Fedora-related software that requires Python 2 to be installed for Fedora to run then by all means install Python 2.  But, if you believe that Python 2 must be installed on Ubuntu Xenial for Ansible playbooks to run there then you are mistaken.  (The `ansible_python_interpreter` technique is used with `VTUL/InstallScripts`'s `Vagrantfile` when Xenial is the base OS.)

Installing unnecessary software is unnecessary.

Reply:
We still use python2 in our current local development, to have only python3 installed is not applied to us. It is out of the scope of this PR. Based on your input, this PR is updated to install Python 3 and added a note to indicate this in the readme.